Documentation/SystemRequirements/PHP.rst.txt

```diff
@@ -45,6 +45,7 @@ Required Extensions
 * **filter**
 * **SPL**
 * **standard**
+* **mbstring** (optional, but a huge performance boost)
```

Comment:
IMHO it should be moved to the section below (like fileinfo, gd, ...).

Reply:
done